src/txs/withdraw.ts

```diff
@@ -8,6 +8,7 @@ export const initiateWithdrawal = async (
   walletClient: L2WalletClient
 ) => {
   const args = await parentClient.buildInitiateWithdrawal({
+    gas: 21_000n,
```

Comment:
This seems to be the only change that's correlated to the error you mention in the PR title. This value is intentionally omitted so that the end user's wallet can supply its own gas value. If that's not happening, that seems like a wallet-specific issue. Can you elaborate on how you're running into the issue?

Reply:
Yes, the problem is that if you don't set gas value (or 0 if it is) then the transaction will be called. This is the problem.
![image](https://github.com/JoinOrigami/custom-gas-token-bridge/assets/106298826/4e2e723b-ad0e-4fa4-9612-4f4187e3b9ec)


Yes, and you can call merge as you see fit.

Reply:
![image](https://github.com/JoinOrigami/custom-gas-token-bridge/assets/106298826/5264fb45-54b2-4607-b0e7-dd656d419188)

Reply:
I believe this is a wallet issue. I'm not running into this when depositing from Base to a custom L3 using an injected browser wallet like MetaMask.